ConeSearch.tex

```diff
@@ -1,13 +1,9 @@
-\documentclass[11pt,a4paper]{ivoa}
-\input tthdefs
+\documentclass[11pt,a4paper]{ivoa} \input tthdefs
```

Comment:
Were the line mergers of multiple commands at the top intended?

Reply:
fixed at commit 401e96c